sdks/python/apache_beam/coders/stream.pyx

```diff
@@ -183,15 +192,16 @@ cdef class InputStream(object):
 
   cpdef libc.stdint.int64_t read_var_int64(self) except? -1:
     """Decode a variable-length encoded long from a stream."""
-    cdef long byte
+    # Inline common case.
+    cdef long byte = <unsigned char> self.allc[self.pos]
+    self.pos += 1
+    if byte < 0x7F:
```

Comment:
Can you explain this condition?

Reply:
Does this represent we need to read the next byte in the stream?

Reply:
Integers that fit into seven bits are stored as single bytes on the stream.

Reply:
thanks